### PR TITLE
Added knowledge library

### PR DESCRIPTION
**Pull request by HarikaMunipalle**

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -61,16 +61,17 @@ <h2 class="text-primary" style="margin-bottom: 9px;font-size: 18px;margin-top: 4
...
-                    <li class="usa-footer__secondary-link"><a  href="{{ site.baseurl}}/video-center/">Learn About Shared Services</a></li>
+                      <li class="usa-footer__secondary-link"> <a href="{{ site.baseurl}}/shared-services-lessons-learned">Knowledge Library</a></li>
+                      <li class="usa-footer__secondary-link"> <a href="{{ site.baseurl}}/assets/files/PMA Placemat.pdf">Advancing the PMA through Shared Services</a></li>
+                      <li class="usa-footer__secondary-link"> <a href="https://www.whitehouse.gov/wp-content/uploads/2019/04/M-19-16.pdf">OMB Memo 19-16</a></li>
+                      <li class="usa-footer__secondary-link"> <a href="{{ site.baseurl }}/assets/files/SQS-Public-FAQs-April-2020.pdf">Frequently Asked Questions</a></li>
+                      <li class="usa-footer__secondary-link"><a href="{{ site.baseurl}}/assets/files/QSMO-Long-term-Designation-Criteria.pdf">Designation Criteria</a></li>
+                      <!--                    <li class="usa-footer__secondary-link"><a href="{{ site.baseurl}}/assets/files/Workforce-Strategies-Guide-4-11-19.pdf">Workforce Strategies Guide</a></li>-->
```
Comment (alexdntowe) [inline _includes/footer.html:69]: is it needed ? if not it can be removed

File: _includes/footer.html [hunk 2]
```diff
@@ -61,16 +61,17 @@ <h2 class="text-primary" style="margin-bottom: 9px;font-size: 18px;margin-top: 4
 
                   <h2 class="text-primary" style="margin-bottom: 9px;font-size: 18px;margin-top: 48px;">Resources, Guidance, and Lessons Learned</h2>
```
Comment (alexdntowe) [inline _includes/footer.html:62]: please add feature on the branch name

File: _includes/footer.html (hunk 1 above)
Comment (HarikaMunipalle) [reply _includes/footer.html]: Sure will do that

File: _includes/footer.html (hunk 2 above)
Comment (HarikaMunipalle) [reply _includes/footer.html:62]: Sure will do that

**Pull request closed without merging**